src/auth/useAuth.ts

```diff
@@ -1,6 +1,7 @@
 import { clearAuthData, getAuthData, saveAuthData } from "@/auth/auth-storage";
 import { useAuthStore } from "@/auth/auth-store";
 import { useEffect } from "react";
+import * as Crypto from 'expo-crypto';
 
 export const useAuth = () => {
```

Comment:
I'm confused, why this useState isn't used for the current authentification/sign in page and there is a specific logic made in https://github.com/info-telecom-strasbourg/InsidePSBS/blob/c94a753b0f0e31137c1f9037c77684d2bb817545/app/auth/_features/sign-in.query.ts#L1
? @romainbourdain